src/net_processing.cpp

```diff
@@ -2443,7 +2443,7 @@ void ProcessMessage(
         vRecv >> vAddr;
 
         // Don't want addr from older versions unless seeding
-        if (pfrom.nVersion < CADDR_TIME_VERSION && connman->GetAddressCount() > 1000)
+        if (pfrom.nVersion < CADDR_TIME_VERSION && connman->GetAddressCount() > ADDRMAN_GETADDR_MAX)
```

Comment:
I think it'd be more appropriate to use `MAX_ADDR_TO_SEND` as the constant here (and below)